renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -3538,6 +3619,18 @@ void WrappedOpenGL::glClearNamedFramebufferfv(GLuint framebuffer, GLenum buffer,
 {
   CoherentMapImplicitBarrier();
 
+  if(IsBackgroundCapturing(m_State))
+  {
+    GLRenderState state;
+    state.MarkDirty(this);
+  }
+  else if(IsActiveCapturing(m_State))
+  {
+    GLRenderState state;
+    state.FetchState(this);
+    state.MarkReferenced(this, false);
```

Comment:
This is a problem with the original function, but these `glClearNamedFramebuffer` functions need to include the FBO with `MarkFBOReferenced` since it's not the bound framebuffer being cleared.

Reply:
I think we also need to add this same thing below into glClearBufferv. At the moment it doesn't have any write references and relies on FBO binding/updating to catch, which is not watertight.

The specific issue I found while testing is that textures bound to FBOs and then cleared with `glClearBufferv` don't have the proper write-reference tracking and get caught at the last minute in one of the FBO bind/update functions.